CHANGELOG.md

```diff
@@ -0,0 +1,57 @@
+# Changelog
+
+All notable changes to the Carbon Aware SDK will be documented in this file.
+
+## [Unreleased]
+
+### Added 
+
+- Added Electricity Map support for forecasting.  This supports the commercial Electricity Map API only, and currently does not support the free Electrity Map API due to differences in the API's.
+
+### Fixed
+
+- Fixed generated test data that had time bomb bug that was in test data, which caused integration tests to fail.  This is now automatically generated each time.
+
+### Changed
+
+#### API
+
+- No API changes have been made
```

Comment:
`/locations` endpoint added in v1.1

Reply:
Yeah, I'll add this one too, it's come after I created this originally :)